tldr.py

```diff
@@ -478,6 +478,13 @@ def emphasise_example(x: str) -> str:
             line = line.replace(r'\{\{', '__ESCAPED_OPEN__')
             line = line.replace(r'\}\}', '__ESCAPED_CLOSE__')
 
+            # Extract long or short options from placeholders
+            if not (shortform and longform):
```

Comment:
Default would obiously be True for both so that this condition causes a skip. I'm not certain what you're asking in the rest of the comment.